CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-cmake_minimum_required(VERSION 3.5)
-if(POLICY CMP0063)
-  cmake_policy(SET CMP0063 NEW)
+cmake_minimum_required(VERSION 3.12.4)
```

Comment:
what's the reason for the version bump?

Reply:
For full support of interface libraries (install, export, target_sources...). This wasn't working properly in previous CMake versions.